### ISSUE TITLE
Can't find the pom.xml file

### ISSUE THREAD

**Issue by mat-ale** [kelseyhightower/nocode#1967]

**Comment by antonsmyk** [comment]
Yes that's a really serious one. This prevents the project artifact to be properly published on Maven Central. How are we supposed to make use of it then??

**Comment by rakeshjuyal** [comment]
@antonsmyk  Please try this command. 
```
```
It worked for me.
